src/js/views/Reports/Components.jsx

```diff
@@ -29,8 +29,13 @@ function Components() {
       )
     }
     if (filterText.length > 0) {
+      const filterColumn = filterText.startsWith('pkg:')
+        ? 'package_url'
+        : 'name'
+      if (filterText.startsWith('pkg:')) {
+      }
```

Comment:
`if` without a body?

Reply:
ah ... leftover from before I switched from an `if` statement to a ternary operator